packages/runtime/container-runtime/src/deltaScheduler.ts

```diff
@@ -101,6 +108,25 @@ export class DeltaScheduler {
                     this.schedulingLog.numberOfTurns++;
                     this.schedulingLog.totalProcessingTime += elapsedTime;
                 }
+
+                setTimeout(() => {
+                    if (this.schedulingLog) {
+                        this.logger.sendTelemetryEvent({
+                            eventName: "InboundOpsPartialProcessingTime",
+                            duration: TelemetryLogger.formatTick(elapsedTime),
+                            opsProcessed: this.schedulingLog.lastSequenceNumber -
+                                this.schedulingLog.firstSequenceNumber,
+                            opsRemainingToProcess: this.deltaManager.inbound.length,
```

Comment:
opsRemainingToProcess should be always zero here, as "idle" event is fired by queue only when there are no more elements in the queue. I'd remove it from logging here.

Reply:
This is not on idle event but before we resume the inbound queue. I was assuming this could be non-zero here. Isn't it the case ?

Reply:
Yes, fine here, comment should have been for InboundOpsProcessingTime event :) There it should be always zero